bats/index.bats

```diff
@@ -0,0 +1,1704 @@
+SQL
+    # found
+    run dolt sql -q "SELECT * FROM onepk WHERE v1 > 70"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "| 5   | 77 | 53 |" ]] || false
+    [[ "${lines[4]}" = "| 3   | 88 | 52 |" ]] || false
+    [[ "${lines[5]}" = "| 1   | 99 | 51 |" ]] || false
+    [[ "${lines[6]}" = "+-----+----+----+" ]] || false
+    # not found
+    run dolt sql -q "SELECT * FROM onepk WHERE v1 > 111"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "+-----+----+----+" ]] || false
+    # not indexed & found
+    run dolt sql -q "SELECT * FROM onepk WHERE v2 > 53"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "| 2   | 11 | 55 |" ]] || false
+    [[ "${lines[4]}" = "| 4   | 22 | 54 |" ]] || false
+    [[ "${lines[5]}" = "+-----+----+----+" ]] || false
+    # not indexed & not found
+    run dolt sql -q "SELECT * FROM onepk WHERE v2 > 111"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "+-----+----+----+" ]] || false
+    # found
+    run dolt sql -q "SELECT * FROM twopk WHERE v2 > 61 AND v1 > 53"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | pk2 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "| 2   | 11  | 55 | 64 |" ]] || false
+    [[ "${lines[4]}" = "| 4   | 22  | 54 | 65 |" ]] || false
+    [[ "${lines[5]}" = "+-----+-----+----+----+" ]] || false
+    # not found key 1
+    run dolt sql -q "SELECT * FROM twopk WHERE v2 > 111 AND v1 > 53"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | pk2 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "+-----+-----+----+----+" ]] || false
+    # not found key 2
+    run dolt sql -q "SELECT * FROM twopk WHERE v2 > 61 AND v1 > 111"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | pk2 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "+-----+-----+----+----+" ]] || false
+    # not found key mismatch
+    run dolt sql -q "SELECT * FROM twopk WHERE v2 > 64 AND v1 > 54"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | pk2 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "+-----+-----+----+----+" ]] || false
+    # not indexed & found
+    run dolt sql -q "SELECT * FROM twopk WHERE v1 > 53"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | pk2 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "| 2   | 11  | 55 | 64 |" ]] || false
+    [[ "${lines[4]}" = "| 4   | 22  | 54 | 65 |" ]] || false
+    [[ "${lines[5]}" = "+-----+-----+----+----+" ]] || false
+    # not indexed & not found
+    run dolt sql -q "SELECT * FROM twopk WHERE v1 > 111"
+    [ "$status" -eq "0" ]
+    [[ "${lines[1]}" = "| pk1 | pk2 | v1 | v2 |" ]] || false
+    [[ "${lines[3]}" = "+-----+-----+----+----+" ]] || false
+}
+
+@test "index: SELECT < Full Match" {
+    skip "comparison does not yet use indexing"

```

Comment:
Same comment. No reason to skip these, just leave a comment that these tests should work regardless of a working index